src/manifest.rs

```diff
@@ -104,8 +104,9 @@ impl Manifest {
                 toml_edit::Item::Table(t) => Ok(t
                     .get_values()
                     .iter()
-                    .map(|(keys, _val)| keys.iter().map(|k| k.to_string()))
+                    .map(|(keys, _val)| keys.iter().map(|&k| k.to_string()))
```

Comment:
I think the `to_string` here is a bug.  `Key::to_string` renders toml.  What you want is `k.get().to_owned()`.

Reply:
(it works because we've only been testing with strings that work as bare words)

Reply:
I forgot about the existence of `k.get()`. this one also conveniently solves the double allocation. Figured that had to be a solution for this but couldn't find it.  I'll address it right now.